mix.exs

```diff
@@ -41,13 +41,15 @@ defmodule Community.Mixfile do
   defp deps do
     [
       {:cowboy, "~> 1.0"},
+      {:ecto, "~> 2.0.0-rc", [override: true, optional: false, hex: :ecto, manager: :mix]},
       {:envy, "~> 0.0.2"},
-      {:ex_machina, "~> 0.6.1"},
+      {:ex_admin, github: "smpallen99/ex_admin"},
+      {:ex_machina, "~> 0.6.1", only: [:dev, :test]},
```

Comment:
You will likely need to use the beta version of this on master for it to work with Ecto 2.0